docs/src/main/paradox/docs/delta/api/supervision-api.md

```diff
@@ -16,4 +16,39 @@ Request
 :   @@snip [supervision-list.sh](assets/supervision/supervision-list.sh)
 
 Response
-:   @@snip [supervision-list.json](assets/supervision/supervision-list.json)
\ No newline at end of file
+:   @@snip [supervision-list.json](assets/supervision/supervision-list.json)
+
+## Projects health
+
+```
+GET /v1/supervision/projects
+```
+
+Indicates whether there are any unhealthy projects.
```

Comment:
We can add a definition of an healthy project (a project that has been correctly provisioned on creation)